caravel/controllers/forms.py

```diff
@@ -33,11 +33,11 @@ class ListingForm(flask_wtf.Form):
     
 
 class NewListingForm(ListingForm):
-    principal = PrincipalField("Seller",
-        validators=[DataRequired(), OnlyValid()])
+    principal = PrincipalField("Seller", validators=[DataRequired()])
```

Comment:
Well, then any email is allowed- but people entering a UChicago email without Shibboleth validation get flagged for moderation. From the user's perspective, then, they are encouraged to sign in simply though expediency.